gencrud/generators/python.py

```diff
@@ -34,17 +34,19 @@
 
 logger = logging.getLogger()
 
-MENU_CHILDEREN_LABEL    = 'children'
-MENU_DISPLAY_NAME       = 'caption'
-MENU_ICON_NAME          = 'icon'
+MENU_CHILDEREN_LABEL    = 'childeren'
```

Comment:
is there a reason why this is now called "childeren" instead of children? This word appears quite often in gencrud/generators/python.py and in gencrud/templates/common/python/main.py

Reply:
children